capture/parsers/udp.c

```diff
@@ -61,15 +61,15 @@ int udp_pre_process(ArkimeSession_t *session, ArkimePacket_t *const packet, int
     packet->direction = (dir &&
                          session->port1 == ntohs(udphdr->uh_sport) &&
                          session->port2 == ntohs(udphdr->uh_dport)) ? 0 : 1;
-    session->databytes[packet->direction] += (packet->pktlen - packet->payloadOffset - 8);
+    session->databytes[packet->direction] += ntohs(udphdr->uh_ulen) - 8;
```

Comment:
I'm not sure why this one should change, since payloadOffset - 8 should be the actual size of the data on the network, or maybe I don't understand.

Reply:
It is correct as long as there isn't any padding after the UDP part, unless I've misunderstood what `session->databytes` represents, I always thought it was the number of bytes that aren't part of the protocol headers.

With the provided PCAP, databytes was 18 instead of 1 before this change.

Reply:
ah ok, so you are saying when there is padding we shouldn't count that as databytes but do could that in totalBytes. That makes sense. Should this also have the same MIN as adding?

Reply:
Yes, exactly.

I'm not sure if it needs the MIN, but could be a good idea just to be safe.